scripts/initialize-build-host.sh

```diff
@@ -288,7 +288,7 @@ then
 
     login="$(cat $HOME/proxy-target.txt)"
 
-    if [ ! -z "$login" ] && $RSH $login true
+    if [ ! -z "$login" ] && $RSH $login true || exit 2 # unstable build can't ssh to build host
```

Comment:
Isn't it better to do it few lines lower, around these lines?
```
            echo "Failed to SSH to proxy target, aborting the build"
	    exit 1
```
just change 1 to 2 here :)

Otherwise, I'm not sure why we would need this block if we `exit` from within if (...) condition ;)